packages/ui/components/core/LocationBasedAlertBanner/index.tsx

```diff
@@ -0,0 +1,80 @@
+import { Box, createStyles, rem, Text } from '@mantine/core'
+import { Trans } from 'next-i18next'
+
+import { type ApiOutput } from '@weareinreach/api'
+import { Link } from '~ui/components/core/Link'
+import { useCustomVariant } from '~ui/hooks/useCustomVariant'
+
+const useStyles = createStyles((theme) => ({
+	banner: {
+		...theme.other.utilityFonts.utility1,
+
+		width: '100vw',
+		height: rem(52),
+		display: 'flex',
+		alignItems: 'center',
+		justifyContent: 'center',
+		textAlign: 'center',
+		position: 'absolute',
+		[theme.fn.largerThan('sm')]: {
+			marginTop: rem(-40),
+		},
+		[theme.fn.largerThan('xl')]: {
+			marginTop: rem(-20),
+		},
+		[theme.fn.smallerThan('sm')]: {
+			height: rem(80),
+		},
+		'&[data-alert-level="INFO"]': {
+			// "Info" level style overrides
+			backgroundColor: theme.other.colors.secondary.cornflower,
+			span: {
+				color: theme.other.colors.secondary.white,
+			},
+		},
+		'&[data-alert-level="WARN"]': {
+			// "Warning" level style overrides
+			backgroundColor: theme.other.colors.tertiary.pink,
+			span: {
+				color: theme.other.colors.secondary.black,
+			},
+		},
+		'&[data-alert-level="CRITICAL"]': {
+			// "Critical" level style overrides
+			backgroundColor: theme.other.colors.tertiary.red,
+			span: {
+				color: theme.other.colors.secondary.white,
+			},
+		},
+	},
+}))
+
+export const LocationBasedAlertBanner = ({
+	i18nKey,
+	ns,
+	defaultText,
+	level,
+}: LocationBasedAlertBannerProps) => {
+	const { classes } = useStyles()
+	const variants = useCustomVariant()
+
+	return (
+		<Box className={classes.banner} data-alert-level={level}>
+			<Text variant={variants.Text.utility1white}>
+				<Trans
+					i18nKey={i18nKey}
+					ns={ns}
+					defaults={defaultText}
+					components={{
+						Link: <Link external variant={variants.Link.inheritStyle} target='_blank'></Link>,
```

Comment:
I ran an experiment and it turns out that you CAN pass through props on components in the translation string. In Storybook, the "Nationwide" example passes in this string: `Test for nationwide alert <Link href="https://www.google.com">with a link</Link>`